modules/guides/pages/bloblang/methods.adoc

```diff
@@ -1867,6 +1866,21 @@ root.id = this.id.string()
 # Out: {"id":"228930314431312345"}
 ```
 
+=== `timestamp`
```

Comment:
Would it be useful to link to this section from https://deploy-preview-127--redpanda-connect.netlify.app/redpanda-connect/guides/bloblang/methods/#timestamp-manipulation?

Reply:
Sure, maybe something like "the output of this method can be transformed further using various timestamp manipulation methods".

Reply:
Updated